packages/atomic/src/components/insight/atomic-insight-result-action-bar/atomic-insight-result-action-bar.pcss

```diff
@@ -0,0 +1,29 @@
+@import '../../../global/global.pcss';
+
+atomic-insight-result-action-bar {
+  display: flex;
```

Comment:
I think a lot of this css can be replaced by tailwind classes which is the preferred way of styling in atomic :)

Reply:
- [ ] tailwind magic